Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Folder/GetIshFolderContent.Tests.ps1

```diff
@@ -77,6 +133,12 @@ Describe “Get-IshFolderContent" -Tags "Read" {
 			#language
 			$ishObjects[0].fstatus.Length -ge 1 | Should Be $true 
 			$ishObjects[0].fstatus_lng_element.StartsWith('VSTATUS') | Should Be $true 
+        }
+        It "version_version_value" { 
```

Comment:
`ishObjects[0].version_version_value` instead of `version_version_value`

Reply:
Done